governance/votes/Executive vote - May 14, 2021.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Template - [Executive Vote] Parameter Changes and Oracle Updates - May 14, 2021
+
+---
+# [Executive Proposal] Parameter Changes and Oracle Updates - May 14, 2021
+
+The Governance Facilitator(s) and the MakerDAO Smart Contracts Domain Team have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://community-development.makerdao.com/en/learn/governance/how-voting-works/) to learn how voting works, and this [wallet setup guide](https://community-development.makerdao.com/en/learn/governance/voting-setup/) to set up your wallet to vote.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+
+### Parameter Changes
+
+Based on the recommendations of the [Open Market Committee](https://forum.makerdao.com/t/parameter-changes-proposal-ppg-omc-001-4-may-2021/7835) the [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee) and [Debt Ceiling](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling/)  parameters will be updated for the vault types below.
+- The Stability Fee for KNC-A, TUSD-A, and PAXUSD-A will be increased.
+- The Stability Fee for ETH-C will be decreased.
+- The Debt Ceiling for KNC-A, PAXUSD-A, and USDC-B will be set to 0.
+
+### Oracle Updates
+
+Based on the recommendations of the Oracle Domain Team, the Uniswap LP Token Oracle will be upgraded with this spell. 
+
+**Voting for this executive proposal will place your MKR in support of the changes outlined above.**
+
+Unless otherwise noted, the changes listed above are subject to the [GSM Pause Delay](https://community-development.makerdao.com/en/learn/governance/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
```

Comment:
Are we sure we need office hours for this?

Reply:
I am not sure haha. My thoughts were that since oracles are changing it should be included incase it were to cause unexpected liquidations